src/main/java/ailurus/AilurusException.java

```diff
@@ -0,0 +1,59 @@
+package ailurus;
+
+public class AilurusException extends IllegalArgumentException {
+    private Error error;
+
+    /**
+     * List of errors for Ailurus.Ailurus
+     */
+    public enum Error {
+        EMPTYLIST, EMPTYTODO, EMPTYEVENT, EMPTYDEADLINE,
+        EMPTYBY, EMPTYAT, BY, AT, NUMBER, NAN, LOAD, FIND,
+    }
+
+    /**
+     * Constructor for Ailurus.AilurusException
+     *
+     * @param e error to be thrown
+     */
+    public AilurusException(Error e) {
+        this.error = e;
+    }
+
+    /**
+     * Overriden method to get error message
+     *
+     * @return error message string
+     */
+    @Override
+    public String getMessage() {
+        switch (this.error) {
```

Comment:
For switch statements, I think there should not be indentation for the case clauses.